compiler/rustc_middle/src/ty/query/plumbing.rs

```diff
@@ -138,6 +150,17 @@ impl<'tcx> TyCtxt<'tcx> {
                 let mut i = 0;
 
                 while let Some(query) = current_query {
+<<<<<<< HEAD
+<<<<<<< HEAD
+                    if backtrace.unwrap() == false && i == num_frames.unwrap() {
+=======
+                    if num_frames == Some(i) {
+>>>>>>> 15827338aa231fd408561bf5db8d8eea85d1a51a
+=======
+                    if num_frames == Some(i) {
+>>>>>>> 15827338aa231fd408561bf5db8d8eea85d1a51a
```

Comment:
This is a messed up rebase.

Reply:
yeah I guess I need close this pull and make the branch again